src/components/Filter/components/FilterMenu/components/FilterSearchField/FilterSearchField.style.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@emotion/react';
+import { rem } from 'theme/utils';
+
+export const textFieldWrapper = () =>
+  css`
+    border-bottom: 1px solid #b9cdfc;
```

Comment:
this doesnt have token?

Reply:
it does but its part of the search tokens. I will add a TODO comment to replace with search tokens when Search component is implemented.